static/css/components/buttonCta.less

```diff
@@ -6,26 +6,36 @@
 // override default anchor styles
 a.cta-btn {
   font-size: 14px;
-  color: @white !important;
```

Comment:
Are you sure you want to drop important here? Can you be sure you haven't broken any of the buttons that use cta-btn but try to override its color?

Reply:
Were these checked, @mekarpeles ?

Reply:
Yes, there's also a follow pr with my better world books changes which addresses other button discrepancies. I feel good about this  as DONE